.github/workflows/centos7-ci.yml

```diff
@@ -29,6 +27,16 @@ jobs:
       with:
         submodules: recursive
 
+    - name: Build rpm package
```

Comment:
Does we really need to build a RPM package each time the master branch or the pull request updating? What about just triggering it when tagging?

Reply:
Yes it would only be triggered when we push to release branch. See the next line
```
if: ${{ startsWith(github.ref, 'refs/heads/release/') }}
```

Reply:
OK, good.